index.html

```diff
@@ -1998,11 +1999,11 @@
           };
         </pre>
         <p>
-          The <a>PaymentRequestUpdateEvent</a> enables the web page to update
+          The <a>PaymentRequestUpdateEvent</a> enables developers to update
           the details of the payment request in response to a user interaction.
         </p>
         <p>
-          If the web page wishes to update the payment request then it should
+          If developers wish to update the payment request then they should
```

Comment:
I feel like this one should be singular, but stick with gender neutral "they". Also, we shouldn't say "should" there... also, wish is kinda vague (I know it's not your wording, but let's fix it now :)). So:

> If a developer wants to update the payment request, then they need to